src/lwc/datatableLWC/datatableLWC.js

```diff
@@ -0,0 +1,40 @@
+import { LightningElement,api,track} from 'lwc';
+import getContacts from '@salesforce/apex/DatatableController.getContacts';
+
+export default class DatatableLwc extends LightningElement {
+@api recordId;
+@track data=[];
+ columns = [
+        { label: 'Contact', fieldName: 'contactName', type: 'text' },
+        { label: 'Phone', fieldName: 'contactPhone',  type: 'phone'},
+        { label: 'Title', fieldName: 'contactTitle',  type: 'text'},
+        { label: 'Account', type: 'accountInfo', fieldName: 'accountName',
+        typeAttributes:
+        {
+            accountId: { fieldName: 'accountId'},
+            accountName: { fieldName: 'accountName'},
+            accountIndustry: { fieldName: 'accountIndustry'},
+            accountRating: { fieldName: 'accountRating'},
+
+                }
+        }
+
+    ];
+
+    connectedCallback(){
+        getContacts({accountId: this.recordId}). then(response => {
+            this.data=response;
+            }).catch(e => {
+                console.log(e);
```

Comment:
This is a good practice to let user some information. It can be a toast with custom message/part of error message/full error message(if have user friendly view)